src/codecTools/createDefaultCodec.ts

```diff
@@ -117,7 +117,7 @@ export const createDefaultCodec = <C extends t.Mixed>(
   }
 
   // The default of an object type is an empty object
-  if (codec instanceof t.ObjectType) {
+  if (codec instanceof t.ObjectType || codec instanceof t.UnknownType) {
```

Comment:
I think this needs to be UnknownRecord, since UnknownType is for `unknown`

Reply:
Interestingly, if I do `codec instanceof t.RecordType` I get error `Right-hand side of 'instanceof' is not callable`